src/app/cli/src/coda_processes.ml

```diff
@@ -9,10 +9,7 @@ let init () = Parallel.init_master ()
 let net_configs n =
   let external_ports = List.init n ~f:(fun i -> 23000 + (i * 2)) in
   let discovery_ports = List.init n ~f:(fun i -> 23000 + 1 + (i * 2)) in
-  let ips =
-    List.init n ~f:(fun i ->
-        Unix.Inet_addr.of_string @@ sprintf "127.0.0.%d" (i + 10) )
-  in
+  let ips = List.init n ~f:(fun _i -> Unix.Inet_addr.of_string "127.0.0.1") in
```

Comment:
This hack existed for the explicit purpose to ensure that each process started locally has a unique ip address so that a single ban entry does not ban all nodes on the network during our integration tests. Why did you undo this?

Reply:
sorry, i am reverting it.